src/libcore/ops.rs

```diff
@@ -64,6 +64,63 @@
 //!
 //! See the documentation for each trait for a minimum implementation that
 //! prints something to the screen.
+//!
+//! The `Fn`, `FnMut`, and `FnOnce` traits are implemented by types that can be
+//! invoked like functions. Note that `Fn` takes `&self`, `FnMut` takes `&mut
+//! self` and `FnOnce` takes `self`. These correspond to the three kinds of
+//! methods that can be invoked on an instance: call-by-reference,
+//! call-by-mutable-reference, and call-by-value. The most common use of these
+//! traits is to act as bounds to higher-level functions that take functions or
+//! closures as arguments.
+//!
+//! Taking a `Fn` as a parameter:
+//!
+//! ```rust
+//! fn call_with_one<F>(func: F) -> usize
+//!     where F: Fn(usize) -> usize
+//! {
+//!     func(1)
+//! }
+//!
+//! let double = |x| x + x;
+//! assert_eq!(call_with_one(double), 2);
+//! ```
+//!
+//! Taking a `FnMut` as a parameter:
+//!
+//! ```rust
+//! fn do_twice<F>(mut func: F, mut arg: &mut usize)
+//!     where F: FnMut(&mut usize)
+//! {
+//!     func(&mut arg);
+//!     func(&mut arg);
+//! }
+//!
+//! let add_two = |x: &mut usize| *x += 2;
```

Comment:
_Oh_. Thanks again for bearing with me while I learn. Third time's a charm, maybe?